superset/assets/javascripts/explore/components/SaveModal.jsx

```diff
@@ -107,7 +107,7 @@ class SaveModal extends React.Component {
       .then((data) => {
         // Go to new slice url or dashboard url
         if (gotodash) {
-          window.location = data.dashboard;
+          window.location = data.dashboard + '&edit=true';
```

Comment:
We might wanna use the `URL` object every where we manipulate URLs to make it more robust, eg:

```javascript
const url = new URL(data.dashboard);
url.searchParams.set('edit', 'true');
window.location = url.href;
```